### PR TITLE
fix paths in docs workflow

### PR DESCRIPTION
**Pull request by pawelru**
follow up on https://github.com/insightsengineering/tlg-catalog/pull/62

### CODE REVIEW

File: book/graphs/pharmacokinetic/pkcg02.qmd [hunk 1]
```diff
@@ -37,7 +37,7 @@ use_footnote1 <- "Program: \nOutput:"
 
 result <- g_ipp(
   df = adpc_a,
-  xvar = "NRELTM1",
+  xvar = "NFRLT",
```
Comment (Melkiades) [inline book/graphs/pharmacokinetic/pkcg02.qmd:40]:
```suggestion
  xvar = "NRRLT ",
```

**Review by Melkiades — APPROVED** [review]
Lgtm! Thanks Pawel ;)

**Review by walkowif — APPROVED** [review]

File: book/tables/pharmacokinetic/adat02.qmd [hunk 2]
```diff
@@ -21,35 +21,42 @@ adab <- synthetic_cdisc_dataset("latest", "adab")
 # Ensure character variables are converted to factors and empty strings and NAs are explicit missing levels.
 adsl <- df_explicit_na(adsl)
 adab <- df_explicit_na(adab) %>%
+  mutate(ADPBLPFL = "Y") %>% # temp fix
```
Comment (pawelru) [inline book/tables/pharmacokinetic/adat02.qmd:24]: I have added a temporary fix to be resolved afterwards

File: book/tables/pharmacokinetic/adat02.qmd [hunk 3]
```diff
@@ -76,7 +83,7 @@ lyt_adab <- basic_table(show_colcounts = TRUE) %>%
 lyt_adab_ti <- basic_table(show_colcounts = TRUE) %>%
   split_cols_by(
     "ACTARM",
-    split_fun = drop_split_levels
+    split_fun = keep_split_levels(c("A: Drug X", "C: Combination", "<Missing>")) # temp fix
```
Comment (pawelru) [inline book/tables/pharmacokinetic/adat02.qmd:86]: I have added a temporary fix to be resolved afterwards

**Pull request merged**